src/frontend/e2e/tests/home.spec.ts

```diff
@@ -0,0 +1,29 @@
+import { expect, test } from "@playwright/test";
+import { getByTestID } from "../utils/selectors";
+
+//todo: populate the remaining elements
+const footer: Record<string, string> = {
+  Github: "https://github.com/chanzuckerberg/czgenepi/",
+  Careers:
+    "https://chanzuckerberg.com/careers/career-opportunities/?initiative=science",
+  "Learning Center":
+    "https://help.czgenepi.org/hc/en-us/categories/6217716150804-Genomic-Epidemiology-Learning-Center",
+};
+
+test.describe("Home page tests", () => {
+  test("Should verify home page", async ({ page }, workerInfo) => {
+    const { baseURL } = workerInfo.config.projects[0].use;
+    await page.goto(`${baseURL}` as string);
+    await expect(page.locator(getByTestID("navbar-landing"))).toBeVisible();
+    await expect(
+      page.locator(getByTestID("navbar-sign-in-link"))
+    ).toBeVisible();
+    await expect(page.locator(getByTestID("logo"))).toBeVisible();
+    Object.keys(footer).forEach((key) => {
+      expect(page.locator(`a:has-text("${key}")`).first()).toHaveAttribute(
```

Comment:
is this an ok selector to have?

Reply:
No they aren't but they are from PoC and have,'t been refactored. I will refactored them once I added test identifiers